src/components/Staking/StakingHomeTableOfContents.tsx

```diff
@@ -33,40 +25,21 @@ const TableOfContentsLink: React.FC<ITableOfContentsLinkProps> = ({
     classes += " active"
   }
   return (
-    <StyledTableOfContentsLink to={url} className={classes}>
+    <Link
+      as={GatsbyLink}
+      to={url}
+      className={classes}
```

Comment:
We can remove this and all its logic with the `isActive` flag. We are not using those classes anymore.